dotcom-rendering/src/components/marketing/epics/ThreeTierChoiceCards.tsx

```diff
@@ -143,11 +166,16 @@ const RecommendedPill = () => {
 	return <div css={recommendedPillStyles}>Recommended</div>;
 };
 
+const DiscountedPill = () => {
+	return <div css={discountedPillStyles}>50% off</div>; //TODO confirm wording on button
```

Comment:
we could derive the copy from the `discount` value here

Reply:
I've added a solution but i'm not sure it's the most elegant